src/Compilers/Core/Portable/DiagnosticAnalyzer/AnalyzerAssemblyLoader.cs

```diff
@@ -59,8 +53,8 @@ internal abstract partial class AnalyzerAssemblyLoader : IAnalyzerAssemblyLoader
         private partial Assembly Load(AssemblyName assemblyName, string assemblyOriginalPath);
 
         /// <summary>
-        /// Determines if the <paramref name="candidateName"/> satisfies the request for 
-        /// <paramref name="requestedName"/>. This is partial'd out as each runtime has a different 
+        /// Determines if the <paramref name="candidateName"/> satisfies the request for
+        /// <paramref name="requestedName"/>. This is partial'd out as each runtime has a different
```

Comment:
These white space changes ... why? Just makes it harder to review.

Reply:
VS Code does it and I don't know how to stop it

Reply:
maybe chat with someone else who uses VS Code and figure it out cause white space changes to lots of files isn't conductive to easy reviews 😄

Reply:
Is trailing whitespace bad? I can find some way to turn it off, but it seems like a good formatting change overall. And "hide whitespace" GH hides it, right?

Reply:
It's unnecessary changes in a PR that we have to take steps to make easier to review. It makes the changes unnecessarily bigger. Generally not nice to include large formatting changes inside of functional changes.